samples/graph-bulk-meetings/csharp/EventMeeting/ClientApp/src/models/dashboard-state.js

```diff
@@ -0,0 +1,7 @@
+const DashboardState = {
+    Default: 'default',
+    View: 'view',
```

Comment:
In the code you are using only default value, remove the other two values

Reply:
Resolved